packages/firestore/src/core/query.ts

```diff
@@ -675,25 +671,25 @@ export class ArrayContainsFilter extends FieldFilter {
 
 /** A Filter that implements the IN operator. */
 export class InFilter extends FieldFilter {
-  constructor(field: FieldPath, value: ArrayValue) {
+  constructor(field: FieldPath, public value: ArrayValue) {
     super(field, Operator.IN, value);
   }
 
   matches(doc: Document): boolean {
-    const arrayValue = this.value as ArrayValue;
+    const arrayValue = this.value;
     const other = doc.field(this.field);
     return other !== undefined && arrayValue.contains(other);
   }
 }
 
 /** A Filter that implements the array-contains-any operator. */
 export class ArrayContainsAnyFilter extends FieldFilter {
-  constructor(field: FieldPath, value: ArrayValue) {
+  constructor(field: FieldPath, public value: ArrayValue) {
     super(field, Operator.ARRAY_CONTAINS_ANY, value);
   }
 
   matches(doc: Document): boolean {
-    const arrayValue = this.value as ArrayValue;
+    const arrayValue = this.value;
```

Comment:
nit: I would just drop the intermediate variables (`arrayValue`) at this point. It is saving 0 characters!